### DISCUSSION TITLE
Is there a way to specify Patroni Tags per cluster member?

### DISCUSSION THREAD

**Discussion by jdutreve** [zalando/postgres-operator#1355] [Q&A]
I would like to specify 2 kinds of replicas : HA sync vs Read-Only async.

**Comment by Jan-M** [comment]
Nope, you can use standby cluster for this. Patroni has no priorties on failover.

**Comment by jdutreve** [comment]
Ok, but will this configuration be applied to these standby replicas ?
configuration : 
   synchronous_node_count: 2 
   synchronous_mode: true
